### PR TITLE
Add assembly constituencies for Jammu & Kashmir union territory.

### PR DESCRIPTION
**Pull request by novitam**
Source: https://en.wikipedia.org/wiki/List_of_constituencies_of_the_Jammu_and_Kashmir_Legislative_Assembly

### CODE REVIEW

**Comment by novitam** [comment]
@jloutsenhizer PTAL, thanks!

File: identifiers/country-in/districts.csv [hunk 1]
```diff
@@ -692,6 +692,26 @@ ocd-division/country:in/territory:dl/district:se,South East Delhi
...
 ocd-division/country:in/territory:dl/district:wd,West Delhi
+ocd-division/country:in/territory:jk/district:an,Anantnag
+ocd-division/country:in/territory:jk/district:bn,Bandipora
+ocd-division/country:in/territory:jk/district:br,Baramulla
+ocd-division/country:in/territory:jk/district:bu,Budgam
+ocd-division/country:in/territory:jk/district:do,Doda
+ocd-division/country:in/territory:jk/district:ga,Ganderbal
+ocd-division/country:in/territory:jk/district:ja,Jammu
+ocd-division/country:in/territory:jk/district:ka,Kathua
+ocd-division/country:in/territory:jk/district:ki,Kishtawar
```
Comment (jloutsenhizer) [inline identifiers/country-in/districts.csv:703]: Should this be spelled `Kishtwar`? https://en.wikipedia.org/wiki/Kishtwar_district

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
Just one comment on a potential spelling issue. Otherwise this looks good to me

File: identifiers/country-in/districts.csv (hunk 1 above)
Comment (novitam) [reply identifiers/country-in/districts.csv]: You are right! Corrected.

**Comment by novitam** [comment]
@jpmckinney can you please review? thanks!

**Review by jpmckinney — APPROVED** [review]
The change in sort order makes this very difficult to review. I'll just assume it's correct. In figure, try to separate sorting into a separate commit, so that I can ignore sorting-only changes.

**Pull request merged**